### PR TITLE
chore: Setup code coverage

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: packages/desktop-updater/babel-register.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require('@babel/register')({
```
Comment (ffflorian) [inline packages/desktop-updater/babel-register.js]:
```suggestion
const babelRegister = require('@babel/register');
babelRegister({
```

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,38 @@
 language: node_js
 
 node_js:
-- '10'
+  - '11'
```
[the remark this answers is not in the document]
Comment (bennyn) [reply .travis.yml]: Yes, because we already use `@types/node` with version 11.

[the remark this answers is not in the document]
Comment (pwnsdx) [reply .travis.yml]: Electron 4 is using Node 10 and Electron 5 is using Node 12, maybe we should use Node 12 then?

**Comment by bennyn** [comment]
@ffflorian `HTTPServer.test.js` was indeed just an empty file that lingered around.

**Pull request merged**